components_library/boards/lupv_board.py

```diff
@@ -111,11 +115,13 @@ def __init__(
         self.lupio_tmr = LupioTMR(pio_addr=0x20006000)
 
         # LUPIO TTY
-        self.lupio_tty = LupioTTY(pio_addr=0x20007000, pic = self.lupio_pic)
+        self.lupio_tty = LupioTTY(pio_addr=0x20007000, platform = self.lupv,
+                                    int_id = 0xa)
```

Comment:
It'd be great to create a dictionary for the irq number or something, instead of using these hardcoded values that can only create bugs down the line.

```py
int_ids = { 'TTY': 0, 'BLK': 1, 'RNG': 2}
...
self.lupio_tty = LupioTTY(pio_addr=0x20007000, platform = self.lupv, int_id = int_ids['TTY'] )
```